tests/pebble.rs

```diff
@@ -63,8 +63,28 @@ async fn http_01() -> Result<(), Box<dyn StdError>> {
     .await?;
     info!(account_id = account.id(), "created ACME account");
 
-    // Issue a certificate w/ HTTP-01 challenge and verify it.
-    pebble.test_http1(&mut account).await?;
+    // Issue a certificate w/ HTTP-01 challenge.
+    let (identifiers, cert_chain) = pebble.test_http1(&mut account).await?;
+
+    // Split off and parse the EE cert, save the intermediates that follow.
```

Comment:
I think putting this in the test directly is going to lead to duplication for the other challenge types which will want to verify the issued certificate the same way. Maybe lift to a helper fn?

Reply:
I did consider that. I think it would make sense to lift to a helper `fn` as soon as we actually have more challenge types, but I felt about about keeping the validation in `issue_certificate()` and it seems okay to keep it here until we add multiple challenge types?